### PR TITLE
DCS-406 Splitting out incident and review routes

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: server/routes/incidents.test.js [hunk 1]
```diff
@@ -94,68 +58,4 @@ describe('GET /your-report', () => {
         expect(res.text).toContain('Use of force report')
       })
   })
-
-  it('should fail to render page when no report', () => {
```
Comment (brightonsbox) [inline server/routes/incidents.test.js:98]: Do you not want to keep this test in for `/your-report`? This is now only testing the happy path for this endpoint now isn't it?

File: server/routes/reviewer.test.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    reviewService.getReport.mockReturnValue({ id: 1, form: { incidentDetails: {} } })
+    return request(app)
+      .get('/1/view-report')
+      .expect(200)
+      .expect('Content-Type', /html/)
+      .expect(res => {
+        expect(res.text).toContain('Use of force report')
+        expect(res.text).not.toContain('Delete report')
+      })
+  })
```
Comment (brightonsbox) [inline server/routes/reviewer.test.js:67]: Is this missing the `should fail to render page when no report` test?

File: server/routes/incidents.test.js (hunk 1 above)
Comment (andrewrlee) [reply server/routes/incidents.test.js:98]: the logic has been pushed down to the service - so we don't explicitly need to test it at this level, its just an unexpected error

File: server/routes/reviewer.test.js (hunk 2 above)
Comment (andrewrlee) [reply server/routes/reviewer.test.js:67]: the logic has been pushed down to the service - so we don't explicitly need to test it at this level, its just an unexpected error

**Pull request merged**